app/controllers/events/view/sessions/list.js

```diff
@@ -1,6 +1,15 @@
 import Controller from '@ember/controller';
+import { computed } from '@ember/object';
 
 export default Controller.extend({
+  ratedSessions: computed('model.feedbacks.@each.session', function() {
+    let ratedSessions = [];
+    this.model.feedbacks.forEach(feedback => {
+      let { session } = feedback;
+      ratedSessions.push(session);
+    });
+    return ratedSessions;
```

Comment:
This is can simplified with [`mapBy`](http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fcomputed/mapBy) ...

```js
ratedSessions: mapBy('model.feedbacks', 'session')
```

Reply:
@niranjan94 I resumed my work on this PR. I have one query that ratedSessions is of `ArrayProxy` type and so cannot be directly used in template. Also I cannot use await here directly. Can you please point out how can this proxy be resolved?

Reply:
array proxy can be directly used as an array ... that's the whole point of an array proxy.